airbyte-integrations/connectors/source-amazon-ads/acceptance-test-docker.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 # Build latest connector image
-docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2)
```

Comment:
why that change?

Reply:
1. 
```
echo connector_image: airbyte/source-amazon-ads:dev | cut -d: -f2
 airbyte/source-amazon-ads
```
2. 
```
echo connector_image: airbyte/source-amazon-ads:dev | cut -d: -f2-
 airbyte/source-amazon-ads:dev
```

if you run connector generator it will generate 2nd variant